### ISSUE TITLE
LongHelp fails to parse help text for old subcommands

### ISSUE THREAD

**Issue by tomatopeel** [ipfs/go-ipfs-cmds#15]
https://github.com/ipfs/go-ipfs-cmds/blob/master/cli/helptext.go#L127

```
$ ipfs pin --help
USAGE
  ipfs pin - Pin (and unpin) objects to local storage.

SYNOPSIS
  ipfs pin
```

**Comment by tomatopeel** [comment]
The cause is that the `subcommandText` function only iterates the `.Subcommands` key of the command, it should test presence of Subcommands and if not present then fall back to OldSubcommands I guess, and return an error if neither present.

https://github.com/ipfs/go-ipfs-cmds/blob/master/cli/helptext.go#L350-L382

**Comment by keks** [comment]
Thank you for the report! Sorry for not responding. I fixed this back in 27a9173da5ab4a18c5066f592eb5432ca1a5db34.
